tests/integration/PipelineMicroserviceCLI/Commands/IntegrationCommandTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+namespace PipelinesMicroserviceCLI\Commands;
+
+use Symfony\Component\Yaml\Yaml;
+
+class IntegrationCommandTest extends \PipelineMicroserviceCLITestCase
+{
+    protected $env = "integration-test";
+    
+    private $db;
+    
+    public function setUp()
+    {
```

Comment:
remove dependency that should not be here, the responsibility of the database belongs to the pipeline-microservice and not to the CLI. Refactor by creating methods to interact directly with the pipeline-microservice to setup the integration tests